{"uuid":"bf5281a0-1424-4e75-8a98-6a951782628e","historyId":"c59643030e1e776644fd5d1d6aa79cfc","status":"failed","statusDetails":{"message":"The following error originated from your test code, not from Cypress.\n\n  > Expected to find a global registry (this usually means you are trying to define steps or hooks in support/e2e.js, which is not supported) (this might be a bug, please report at https://github.com/badeball/cypress-cucumber-preprocessor)\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.","trace":"Error: The following error originated from your test code, not from Cypress.\n\n  > Expected to find a global registry (this usually means you are trying to define steps or hooks in support/e2e.js, which is not supported) (this might be a bug, please report at https://github.com/badeball/cypress-cucumber-preprocessor)\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.\n    at createError (http://localhost:8000/__cypress/tests?p=cypress/support/e2e.js:13376:16)\n    at fail (http://localhost:8000/__cypress/tests?p=cypress/support/e2e.js:13380:15)\n    at assert (http://localhost:8000/__cypress/tests?p=cypress/support/e2e.js:13387:9)\n    at assertAndReturn (http://localhost:8000/__cypress/tests?p=cypress/support/e2e.js:13391:9)\n    at getRegistry (http://localhost:8000/__cypress/tests?p=cypress/support/e2e.js:22411:49)\n    at defineStep (http://localhost:8000/__cypress/tests?p=cypress/support/e2e.js:23433:36)\n    at cypress/e2e/step_definitions/common/auth_steps.js (http://localhost:8000/__cypress/tests?p=cypress/support/e2e.js:23558:54)\n    at __init (http://localhost:8000/__cypress/tests?p=cypress/support/e2e.js:17:58)\n    at eval (http://localhost:8000/__cypress/tests?p=cypress/support/e2e.js:24758:3)\n    at eval (http://localhost:8000/__cypress/tests?p=cypress/support/e2e.js:24808:3)"},"stage":"finished","steps":[],"attachments":[{"name":"cypress/e2e/features/assignment_viewing.feature:2025-05-03T14:31:09.127Z","type":"image/png","source":"0de06f46-b0db-4351-a5d6-f5e0dd87e546-attachment.png"},{"name":"video recording","type":"video/mp4","source":"bdaa4835-9324-4fff-bc6e-fb7153c09832-attachment.mp4"}],"parameters":[],"labels":[{"name":"package","value":"cypress.e2e.features.assignment_viewing.feature"}],"links":[],"start":1746282669123,"name":"An uncaught error was detected outside of a test","fullName":"An uncaught error was detected outside of a test","stop":1746282669361}